geomesa-stream/geomesa-stream-geoserver-plugin/src/main/scala/org/locationtech/geomesa/stream/plugin/StreamDataStoreEditPanel.scala

```diff
@@ -0,0 +1,94 @@
+/*
+ * Copyright 2014 Commonwealth Computer Research, Inc.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.locationtech.geomesa.stream.plugin
+
+import org.apache.wicket.behavior.SimpleAttributeModifier
+import org.apache.wicket.markup.html.form.validation.IFormValidator
+import org.apache.wicket.markup.html.form.{Form, FormComponent}
+import org.apache.wicket.markup.html.panel.Panel
+import org.apache.wicket.model.{IModel, PropertyModel, ResourceModel}
+import org.geoserver.web.data.store.StoreEditPanel
+import org.geoserver.web.data.store.panel.{ParamPanel, PasswordParamPanel, TextAreaParamPanel, TextParamPanel}
+import org.geoserver.web.util.MapModel
+import org.geotools.data.DataAccessFactory.Param
+import org.locationtech.geomesa.stream.datastore.StreamDataStoreParams
+
+
+class StreamDataStoreEditPanel(componentId: String, storeEditForm: Form[_])
```

Comment:
Is it possible to get text areas from the params/hints in the ds config? we're going to have to remove all these custom wicket pages at some point for CQ reasons. Maybe we should provide a patch to geotools/geoserver to handle text areas if it's not already there.